app/controllers/memberships/join_zusatzsektions_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Memberships
+  class JoinZusatzsektionsController < Wizards::BaseController
+    before_action :wizard, :person, :group, :authorize
+
+    helper_method :group, :person
+    alias entry wizard
+
+    private
+
+    def authorize
+      authorize!(:create, wizard)
+    end
+
+    def wizard
+      @wizard ||= model_class.new(
+        person: person,
+        current_step: params[:step].to_i,
+        backoffice: person.backoffice?,
```

Comment:
This should depend on the current logged in user, not the person to be edited, right?